mgmt_cli_test.py

```diff
@@ -89,21 +89,11 @@ def install_awscli_dependencies(self, node):
         self._run_cmd_with_retry(executor=node.remoter.sudo, cmd=shell_script_cmd(cmd))
 
     def install_gsutil_dependencies(self, node):
-        def is_dir_exists(path):
-            shell = dedent(f"""
-            if [ -d {path} ]
-            then
-                exit 0
-            else
-                exit 1
-            fi
-            """)
-            result = node.remoter.sudo(shell, ignore_status=True)
+        def is_gsutil_installed():
+            result = node.remoter.run("gsutil -v", ignore_status=True)
             return result.exited == 0
-
-        sdk_directory_path = '$HOME/google-cloud-sdk'
-        if not is_dir_exists(path=sdk_directory_path):
-            self._run_cmd_with_retry(executor=node.remoter.run, cmd=shell_script_cmd("""\
+        if not is_gsutil_installed():
+            self._run_cmd_with_retry(executor=node.remoter.sudo, cmd=shell_script_cmd("""\
```

Comment:
from a quick check i did on live nodes related to this fix, this script doesn't work, as `gsutil` is not installed in none of the default paths set in the `PATH` environment variable...